File: healio/ai-engine/utils/logger.py
from loguru import logger
import sys

logger.remove()
logger.add(sys.stderr, level='INFO', format='{time:YYYY-MM-DD HH:mm:ss} | {level} | {message}')
logger.add('logs/ai_engine.log', rotation='10 MB', retention='30 days', level='DEBUG')

def get_logger():
    return logger

File: healio/frontend/src/utils/logger.js
const isDev = __DEV__;
export const logger = {
  info: (...args) => isDev && console.log('[INFO]', ...args),
  warn: (...args) => isDev && console.warn('[WARN]', ...args),
  error: (...args) => console.error('[ERROR]', ...args),
  debug: (...args) => isDev && console.debug('[DEBUG]', ...args),
};
